src/Component/CovidGraph/CovidGraph.jsx

```diff
@@ -8,54 +9,66 @@ import {
   Tooltip,
   Area,
 } from "recharts";
+
+
 export function MyCovidApp() {
+  
   const [coviddata, setCovidData] = useState([]);
-
+// let obj =[{
+//   country:"india",
+//   TotalCases:"983493",
+//   TotalDeaths:"32490"
+// }]
   useEffect(() => {
```

Comment:
Don't call the data here directly. Call the data in the other component where you import the chart and pass data as props.

Reply:
okay sir